scripts/healthcheck.sh

```diff
@@ -2,9 +2,9 @@
 set -ex
 cd "$(dirname "$0")"
 source ./common.sh
+mount_boards
 
-# reuse checkout
-cd ../..
+cd
```

Comment:
Is it guaranteed that `HOME` is always set ?

Reply:
I don't know but it works currently. Do you want me to `cd /` instead?

Reply:
No, that is ok I suppose.